### PR TITLE
Separate setting providers credentials.

### PR DESCRIPTION
**Pull request by nedSaf**
#137

### CODE REVIEW

**Comment by nedSaf** [comment]
@amitaibu PR is ready
account

File: client/app/scripts/controllers/account.js [hunk 1]
```diff
@@ -24,16 +24,19 @@ angular.module('clientApp')
...
+     *  The type of the provider credentials to save.
      */
-    $scope.saveAccountData = function() {
-      var data = {
-        'browserstack_username': $scope.account.browserstack_username,
-        'browserstack_key': $scope.account.browserstack_key,
-        'sauce_username': $scope.account.sauce_username,
-        'sauce_access_key': $scope.account.sauce_access_key
-      };
+    $scope.saveAccountData = function(type) {
```
Comment (amitaibu) [inline client/app/scripts/controllers/account.js]: saveAccountData => SaveSeleniumProvidersData

**Comment by nedSaf** [comment]
@amitaibu Ready.

**Comment by amitaibu** [comment]
Merged.

**Pull request closed without merging**